tabbing/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	init: function (flags) {
+
+		if (flags.get('a11yKeyboardFocus')) {
+			const container = document.querySelector('.js');
+
+			window.addEventListener('keyup', function (e) {
```

Comment:
If we did end up inlining this code we’d have to handle `attachEvent/detachEvent`, as well as avoiding ES6 bits as you mentioned offline